packages/manager/modules/nasha/src/components/metrics/metrics.component.js

```diff
@@ -0,0 +1,12 @@
+import template from './metrics.template.html';
+
+export default {
+  bindings: {
+    isMonitoredDisabled: '< monitoredDisabled',
```

Comment:
```suggestion
    isMonitoredDisabled: '<',
```
**suggestion**: better to use the same name

Reply:
:+1: